UltraStar Play/Assets/Scenes/PartyMode/PartyModeSettings/DefaultPartyModeSettingsSceneDataProvider.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Linq;
+using UnityEngine;
+
+public class DefaultPartyModeSettingsSceneDataProvider : MonoBehaviour, IDefaultSceneDataProvider
```

Comment:
- Good that you added default scene data
- Every scene should be playable / testable in the Unity editor. However, PartyModeVersusScene has exceptions.